Softeq.XToolkit.Common/Helpers/TextRange.cs

```diff
@@ -1,25 +1,62 @@
 ﻿// Developed by Softeq Development Corporation
 // http://www.softeq.com
 
+using System;
+
 namespace Softeq.XToolkit.Common.Helpers
 {
+    /// <summary>
+    /// A class containing information about a range of text (it's start position and length)
+    /// </summary>
     public class TextRange
     {
+        /// <summary>
+        /// Creates a text range. Parameters can not be negative
+        /// </summary>
+        /// <param name="position">Start position of this range</param>
+        /// <param name="length">Length of this range</param>
         public TextRange(int position, int length)
         {
+            if (position < 0 || length < 0)
```

Comment:
`ArgumentOutOfRangeException` can be better

Reply:
Agree. Fixed